packages/core/src/behaviors.ts

```diff
@@ -172,6 +161,12 @@ export function spawnBehavior<TEvent extends EventObject, TEmitted>(
         }
       };
     },
+    start() {
+      mailbox.start();
+    },
+    stop() {
+      mailbox.clear();
+    },
```

Comment:
I think there is an urgent need to define "primitive types" of XState. Quite frankly, I'm getting lost in that - there are a lot of similar, yet slightly different abstractions introduced in the codebase and it's hard to manage their responsibilities because they are not entirely well-defined. I think that I understand most of them but yet the implementation doesn't mirror what I assume and this is confusing.

Reply:
Can you list exactly what's confusing?

Reply:
Basically what we've discussed last week - atm behaviors are not so pure but they should be. Whatever responsibilities are managed by behaviors should be pulled into actors.